pkg/crc/constants/constants.go

```diff
@@ -48,6 +49,16 @@ func GetOcUrl() string {
 	return GetOcUrlForOs(runtime.GOOS)
 }
 
+var PodmanUrlForOs = map[string]string{
+	"darwin":  fmt.Sprintf("%s/%s", DefaultPodmanUrlBase, "podman-remote-latest-master-darwin-amd64.zip"),
+	"linux":   fmt.Sprintf("%s/%s", DefaultPodmanUrlBase, "podman-remote-latest-master-linux---amd64.zip"),
+	"windows": fmt.Sprintf("%s/%s", DefaultPodmanUrlBase, "podman-remote-latest-master-windows-amd64.msi"),
```

Comment:
Why are we using the msi ?

Reply:
I also created #1061 as the below example is not a good practice. This can be done without the `map` and `runtime.GOOS` selection by moving these file identifiers to the respective `_darwin.go`, `_windows.go`, etc files and just use as `PodmanUrl` instead.

Reply:
> Why are we using the msi ?

Copy&paste error. Will fix.